### PR TITLE
support macOS from 10.15

### PR DESCRIPTION
**Pull request by oozoofrog**
#151 

- add support macOS 10.15
- support AppKit and UIKit both!
- update run-test.sh for iOS, macOS both!

스크린샷 2022-03-12 오전 12 16 39

nef-exportSnippetToFile 2022-03-12 00 17 04

### CODE REVIEW

File: Sources/SwiftLayout/Extension/Kit/ViewAlias.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  
+//
+//  Created by oozoofrog on 2022/03/11.
+//
+
+import Foundation
+
+#if canImport(AppKit)
+import AppKit
+public typealias UIView = AppKit.NSView
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/Kit/ViewAlias.swift:12]: It doesn't seem like a good idea to typealias `NSView` directly to `UIView`.\
In my opinion, it is better to define a new type like `SLView` (SwiftLayout View) or `NSUIView`.

If you do this, it is expected that the following imports can also be removed from some files.
```swift
#if canImport(UIKit)
import UIKit
#elseif canImport(AppKit)
import AppKit
#endif
 ```

**Review by gmlwhdtjd — COMMENTED** [review]
I think supporting AppKit is a very good idea!! 👍
However, there are different opinions on implementation, so please check it once.

File: Sources/SwiftLayout/Extension/Kit/ViewAlias.swift (hunk 1 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Kit/ViewAlias.swift:12]: @gmlwhdtjd thank you. you right absolutely. and  have something to confess, there was be SwiftLayoutView on first commit of pr. it was removed by my mistook, and leave from us. so, i swear here on m1 max Mac book (of future), to get that code back from Code Underworld.

Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Kit/ViewAlias.swift]: @gmlwhdtjd show our SL*!

**Review by gmlwhdtjd — APPROVED** [review]
Awesome!!!!👍👍👍👍👍

**Comment by oozoofrog** [comment]
> Awesome!!!!👍👍👍👍👍

thank you!!!! 👍👍👍👍👍

**Pull request merged**